package.json

```diff
@@ -6,7 +6,7 @@
     "test": "tests"
   },
   "scripts": {
-    "test": "bender run",
+    "test": "bender run -l && bender run -b firefox",
```

Comment:
Bender is CKeditor's testing framework.  it can run using either a headless browser or actual browser.  Unfortunately, phantomjs version that it supports is quite old, and runs into random weird issues around selection, sometimes.  We're just trying to get this stuff working for prototyping and user testing purposes, so I opted to not worry about it